### ISSUE TITLE
Specifying 'Org Chart' layout with 'type' and 'spacing' parameters

### ISSUE THREAD

**Issue by exwhyz** [jgraph/drawio#2028]
I am using the below Layout setting as part of Arrange -> Insert -> Advanced -> CSV. Is there a way to specify the layout as 'Org Chart' to mimic the Arrange -> Layout -> Org Chart function in the UI and then be able to specify 'Smart' as the 'Org Chart Type' as well as the 'Parent Child Spacing' and 'Sibling Spacing' parameters? 

I have tried the layout settings below but I am not getting the desired 'Org Chart' layout with 'Smart' type or any other Org Chart types for that matter. I have looked at the api (https://jgraph.github.io/mxgraph/docs/js-api/files/layout/hierarchical/mxHierarchicalLayout-js.html) but it is not clear how to arrange this layout using CSV import settings:

```
# layout: [{"layout": "mxHierarchicalLayout","config": {"orgChartType":"smart", "parentChildSpacing":20, "siblingSpacing":40}}]
```

**Update**: I think I got a step closer with this layout JSON however I am still missing something as the orgChartType property doesn't seem to be working. But I can see the spacing properties working fine when I render the graph:

```
# layout: [{"layout": "mxOrgChartLayout","config": {"orgChartType": "1ColumnRight","parentChildSpacing": 10,"siblingSpacing": 10}}]
```

**Comment by exwhyz** [comment]
I was able to figure out the Org Chart Type parameter that does the trick and this is the Layout JSON that worked for me:

```
# layout: [{"layout": "mxOrgChartLayout", "config": {"branchOptimizer": "branchOptimizerAllSingleColumnRight","parentChildSpacing": 10,"siblingSpacing": 10}}]
```

**Comment by haraldFo** [comment]
what is the parameter that does the trick? I am trying to get the orgcharttype "Hanger 2" but did not find anything in the API ...

**Comment by exwhyz** [comment]
mxOrgChartLayout did the trick but not any more since the new version of draw.io this parameter results in an invalid call error and does not work anymore. On an old version of draw.io this still works as expected.

**Comment by alderg** [comment]
Thanks for the report. This will be fixed in the next release.

**Comment by alderg** [comment]
PS: To get the values for the orgchart layout we'll add a Custom button to the Orgchart dialog.
![Screenshot 2023-01-22 at 07 59 09](https://user-images.githubusercontent.com/921200/213904628-97e9dc82-c966-4a2f-a5a4-43a154d6d03d.png)

**Comment by davidjgraph** [comment]
Internal release https://drawio.atlassian.net/projects/DS/versions/11256
Internal commit https://github.com/jgraph/drawio-dev/commit/119afdc65a6c76e60b56f5a00c6ad638bfb6428a

**Comment by davidjgraph** [comment]
Released in 20.8.11
